wormhole-connect/src/routes/operator.ts

```diff
@@ -369,34 +369,15 @@ export class Operator {
     for (const route of config.routes) {
       const r = this.getRoute(route as Route);
 
-      try {
-        const sourceTokens = await r.supportedSourceTokens(
-          config.tokensArr,
-          destToken,
-          sourceChain,
-          destChain,
-        );
+      const sourceTokens = await r.supportedSourceTokens(
```

Comment:
Does removing the try/catch mean we won't get any exceptions from any of the routes for supported source tokens? If not, maybe leave it just logging purposes?

Reply:
no. this try/catch was only here for a short time in an attempt to gracefully transition to SDKv2, which returns all supported destination tokens in one method call. the catch case was for legacy interface which wouldn't handle that, and instead need to check each token individually. we got rid of the legacy interface completely so this try/catch isn't required anymore.